site2/docs/client-libraries-java.md

```diff
@@ -402,6 +403,25 @@ consumer.acknowledge(messages)
 >     .build();
 > ```
 
+### NegativeAckRedeliveryBackoff
```

Comment:
Does this belong to`Negative Ack` in [this sheet](https://docs.google.com/spreadsheets/d/1YHYTkIXR8-Ql103u-IMI18TXLlGStK8uJjDsOOA0T20/edit#gid=1784579914&range=B73:C73)?
If so, how about changing the heading to `Negative acknowledgment`?